### ISSUE TITLE
Provide mechanisms to combine evolution with local search

### ISSUE THREAD

**Issue by jakobj** [Happy-Algorithms-League/python-gp#59]
In particular a mechanism for the updating of node values from pytorch classes in which parameters have been optimized via gradient descent would be desirable. #57 provides a first stab at this.
